### PR TITLE
ci: bump actions versions

### PR DESCRIPTION
**Pull request by scribam**

### CODE REVIEW

File: .github/workflows/glad2.yaml [hunk 1]
```diff
@@ -31,8 +30,8 @@ jobs:
       - name: Run Tests
         run: PRINT_MESSAGE=1 xvfb-run --auto-servernum ./utility/test.sh
       - name: Publish Test Results
-        uses: EnricoMi/publish-unit-test-result-action@v1
+        uses: EnricoMi/publish-unit-test-result-action@v2
         if: always()
         with:
-          files: test-report.xml
+          junit_files: test-report.xml
```
Comment (Dav1dde) [inline .github/workflows/glad2.yaml:36]: Is this correct, the docs https://github.com/EnricoMi/publish-unit-test-result-action/tree/v2.4.1 still use `files`

Comment (scribam) [reply .github/workflows/glad2.yaml:36]: Indeed, they reintroduced `files` last week https://github.com/EnricoMi/publish-unit-test-result-action/pull/395, I will update this PR accordingly

Comment (scribam) [reply .github/workflows/glad2.yaml]: PR updated

Comment (Dav1dde) [reply .github/workflows/glad2.yaml]: Thank you, that was amazingly fast!

**Pull request merged**